README.md

```diff
@@ -1,82 +1,479 @@
+[![Test](https://github.com/tablelandnetwork/go-tableland/actions/workflows/test.yml/badge.svg)](https://github.com/tablelandnetwork/go-tableland/actions/workflows/test.yml)
+[![Go Reference](https://pkg.go.dev/badge/github.com/textileio/go-tableland.svg)](https://pkg.go.dev/github.com/textileio/go-tableland)
+[![Go Report Card](https://goreportcard.com/badge/github.com/textileio/go-tableland)](https://goreportcard.com/report/github.com/textileio/go-tableland)
+[![License](https://img.shields.io/github/license/tablelandnetwork/go-tableland.svg)](./LICENSE)
+[![Release](https://img.shields.io/github/release/tablelandnetwork/go-tableland.svg)](https://github.com/tablelandnetwork/go-tableland/releases/latest)
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg)](https://github.com/RichardLitt/standard-readme)
+
+> Go implementation of the Tableland database—run your own node, handling on-chain mutating events and serving read-queries.
+
+## Table of Contents
+
+- [Background](#background)
+  - [What is a validator?](#what-is-a-validator)
+  - [Validator and network relationship](#validator-and-network-relationship)
+  - [Running a validator](#running-a-validator)
+- [Usage](#usage)
+  - [System requirements](#system-requirements)
+  - [Firewall configuration](#firewall-configuration)
+  - [System prerequisites](#system-prerequisites)
+  - [Run the validator](#run-the-validator)
+  - [Docker Compose setup](#docker-compose-setup)
+  - [Backups and other routines](#backups-and-other-routines)
+- [Development](#development)
+  - [Configuration](#configuration)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Background
+
+`go-tableland` is a Go language implementation of a Tableland node, enabling developers and service providers to run nodes on the Tableland network and host databases for web3 users and applications. Note that the Tableland protocol is currently in open beta, so node operators have the opportunity to be one of the early network adopters while the responsibilities of the validator will continue to change as the Tableland protocol evolves.
+
+### What is a validator?
 
 Validators are the execution unit/actors of the protocol.
 
 They have the following responsibilities:
 
 - Listen to on-chain events to materialize Tableland-compliant SQL queries in a database engine (currently, SQLite by default).
-- Serve read-queries (e.g: `SELECT * FROM foo_69_1`) to the external world.
+- Serve read-queries (e.g., `SELECT * FROM foo_69_1`) to the external world.
+
+> In the future, validators will have more responsibilities in the network.
+
+### Validator and network relationship
+
+The following diagram describes a high level interaction between the validator, EVM chains, and the external world:
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/13358940/249310798-a65732b9-a48b-4547-bc4d-71af8bd4e09f.png" width='80%'/>
```

Comment:
New diagram looks really neat!

Reply:
thanks! small tweaks to the original that someone else created, but i figured i'd make it a _bit_ prettier lol

Reply:
haha! yeah i guess that was me who made the ugly one

Reply:
Hahah oh...I meant...um, "not ugly" 😅